lib/middleware/bearerToken.ts

```diff
@@ -0,0 +1,53 @@
+import { NextApiRequest, NextApiResponse } from "next";
+import jwt from "jsonwebtoken";
+import { getTokenById } from "../../pages/api/id/[form]/bearer";
```

Comment:
I think we could move this into the crud file in the lib folder rather than importing from another api file. Something for another PR though